src/app/entity-groups/journal-entities/item-pages/journal-issue/journal-issue.component.scss

```diff
@@ -1 +1,5 @@
 @import '../../../../../styles/variables.scss';
+
+button.back-btn {
+  margin-bottom: 10px;
```

Comment:
Perhaps use [a bootstrap `m-` class](https://getbootstrap.com/docs/4.1/utilities/spacing/) rather than setting the margin manually in pixels, that would scale with the theme.
 
And again, if you do that in `ds-results-back-button` you don't have to repeat it for every entity type